orchprovider/k8s/v1/util.go

```diff
@@ -193,6 +193,21 @@ func (p *VolumeMarkerBuilder) AddControllerStatuses(pod k8sApiV1.Pod) {
 	_ = p.AddMultiples(string(v1.JivaControllerStatusVK), status, true)
 }
 
+func (p *VolumeMarkerBuilder) AddControllerContainerStatuses(cp k8sApiV1.Pod) {
```

Comment:
Include a method AddReplicaContainerStatuses

Reply:
There should be a method e.g. `func (p *VolumeMarkerBuilder) IsRunning(keys ...String) bool {...}`

This should check of all the passed keys have `Running` as their values. If any key does not have Running as their value, then it returns false.

Reply:
Above IsRunning() method call will be used to set `Running` or `NotRunning` status to returned Volume object.